### DISCUSSION TITLE
interview createAt updateAt

### DISCUSSION THREAD

**Discussion by her0807** [woowacourse-teams/2022-ternoko#202] [Q&A]
현재 Interview 와 formItem 이 생성된 시간을 관리하는 컬럼이 없음. 추가 하고 싶음!

```java

@Getter
@MappedSuperclass
@EntityListeners(AuditingEntityListener.class)
public abstract class AuditingEntity {

    @Column(nullable = false, updatable = false)
    @CreatedDate
    private LocalDateTime createdAt;

    @LastModifiedDate
    private LocalDateTime updatedAt;

}


```

**Comment by her0807** [comment]
@HyeonbinSa @Juhyung990122 @soominsohn @dongho108

**Comment by Juhyung990122** [comment]
생성된 시간을 필드로 관리해야하는 이유가 궁금합니다~~!

**Comment by soominsohn** [comment]
createdAt과 updatedAt을 넣어서 얻는 장점이 있을까요?
수달은 어떤 것을 관리하고 싶으신지 알고싶어요 😄

**Comment by HyeonbinSa** [comment]
저도 위에 작성해주신 의견과 같이 `사용 목적`이랑 얻을 수 있는 `장점`
그리고 데이터베이스에서 어떻게 구현되는지 궁금합니다. @MappedSuperclass나 @EntityListeners(AuditingEntityListener.class) 처음 접하는 어노테이션이 있어서 간략해서 설명도 해주시면 이해하기 좋을거같습니다.
